nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/nf-connection-details.js

```diff
@@ -521,6 +524,27 @@
                         nfCommon.populateField('read-only-flow-file-expiration', connection.flowFileExpiration);
                         nfCommon.populateField('read-only-back-pressure-object-threshold', connection.backPressureObjectThreshold);
                         nfCommon.populateField('read-only-back-pressure-data-size-threshold', connection.backPressureDataSizeThreshold);
+                        nfCommon.populateField('read-only-load-balance-strategy', nfCommon.getComboOptionText($('#load-balance-strategy-combo'), connection.loadBalanceStrategy));
+                        nfCommon.populateField('read-only-load-balance-partition-attribute', connection.loadBalancePartitionAttribute);
+                        nfCommon.populateField('read-only-load-balance-compression', nfCommon.getComboOptionText($('#load-balance-compression-combo'), connection.loadBalanceCompression));
```

Comment:
I don't think we can access the combo's here and above on line 527. Opening the connection details dialog from the canvas (when the source is running for instance) probably works because the combos exist someplace else on the page. However, opening the connection details dialog from the Summary table fails because those combos do not exist in the DOM.

If you're looking to avoid duplicating code, I would probably move the combo configuration into a shared place and access that from both places. I believe we did something similar with the policies combo.